app/assets/javascripts/discourse/app/instance-initializers/inject-objects.js

```diff
@@ -4,14 +4,13 @@ import Site from "discourse/models/site";
 import deprecated from "discourse-common/lib/deprecated";
 
 export default {
-  name: "inject-objects",
   after: "sniff-capabilities",
 
-  initialize(container, app) {
+  initialize(owner) {
     // This is required for Ember CLI tests to work
-    setDefaultOwner(app.__container__);
+    setDefaultOwner(owner.__container__);
```

Comment:
This is fishy per above, but this is what the existing code does and changing it causes issues